### PR TITLE
bug: typo when using slice when meaning projection

### PR DESCRIPTION
**Pull request by franciscolopezsancho**
From my understanding, this is a typo. Could you confirm?

### CODE REVIEW

File: docs/src/main/paradox/projection.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ for the previous slice distributions.
 A slice is deterministically defined based on the persistence id. The purpose is to evenly distribute all
 persistence ids over the slices. The `eventsBySlices` query is for a range of the slices. For example if
 using 1024 slices and running 4 Projection instances the slice ranges would be 0-255, 256-511, 512-767, 768-1023.
-Changing to 8 slice ranges means that the ranges would be 0-127, 128-255, 256-383, ..., 768-895, 896-1023.
+Changing to 8 projections means that the ranges would be 0-127, 128-255, 256-383, ..., 768-895, 896-1023.
```
Comment (octonato) [inline docs/src/main/paradox/projection.md:90]: what about this

```suggestion
Changing to 8 projections means that the slice ranges would be 0-127, 128-255, 256-383, ..., 768-895, 896-1023.
```

File: docs/src/main/paradox/query.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ for the previous slice distributions.
 A slice is deterministically defined based on the persistence id. The purpose is to evenly distribute all
 persistence ids over the slices. The `eventsBySlices` query is for a range of the slices. For example if
 using 1024 slices and running 4 Projection instances the slice ranges would be 0-255, 256-511, 512-767, 768-1023.
-Changing to 8 slice ranges means that the ranges would be 0-127, 128-255, 256-383, ..., 768-895, 896-1023.
+Changing to 8 projections means that the ranges would be 0-127, 128-255, 256-383, ..., 768-895, 896-1023.
```
Comment (octonato) [inline docs/src/main/paradox/query.md:56]:
```suggestion
Changing to 8 projections means that the slice ranges would be 0-127, 128-255, 256-383, ..., 768-895, 896-1023.
```

**Comment by patriknw** [comment]
This code and documentation has moved to akka-projection, so please open PR there instead.
(it will be removed here)

Also, I wouldn't classify this as a `bug`.

**Comment by octonato** [comment]
> This code and documentation has moved to akka-projection, so please open PR there instead. (it will be removed here)

The paragraph just above it talks about projection. It says: 

> For example if using 1024 slices and running 4 Projection instances the slice ranges would be 0-255, 256-511, 512-767, 768-1023.

**Comment by franciscolopezsancho** [comment]
Roger that (both). I'll move it there.
I'll take those comments @octonato.

**Pull request closed without merging**